core/common/src/main/kotlin/eu/kanade/tachiyomi/network/AndroidCookieJar.kt

```diff
@@ -51,4 +52,40 @@ class AndroidCookieJar : CookieJar {
     fun removeAll() {
         manager.removeAllCookies {}
     }
+
+    fun addAll(url: HttpUrl, cookies: List<Cookie>) {
```

Comment:
Remove debug logs if you are done with them, cookies shouldn't be logged if possible